From db4f98abbbaa105c00c874ecf06e3a0ae9500894 Mon Sep 17 00:00:00 2001
From: Paul Marquess <pmqs@cpan.org>
Date: Sat, 25 Oct 2025 11:34:00 -0400
Subject: [PATCH] cpan/Compress-Raw-Bzip2 - Update to version 2.218

  2.214 24 October 2025

      * version 2.214
        Fri Oct 24 16:15:22 2025 +0100
        5fd4a5676d0e9a283d55fdf7d2a14957292060dd

      * Add dependbot yml file to police the workflow files
        Thu Sep 4 16:57:57 2025 +0100
        af071123f21dbd30cd1c0caab4fd126512007161

      * Add perl 5.42 to workflow files
        Thu Sep 4 16:57:06 2025 +0100
        efd6a1b6cb0e2be7a26747b427575d6aa4bee6d6

  2.217 31 January 2026

      * Update version to 2.217
        Sat Jan 31 11:52:54 2026 +0000
        773773f59082a73134212a2d63368683027122ed

  2.218 8 March 2026

      * Version 2.218
        Sun Mar 8 13:47:17 2026 +0000
        be6054d7ed1536eec4e5cf04117f251fb4389d59

      * Add SECURITY.md Fixes #18
        Sun Mar 8 12:14:23 2026 +0000
        69f9bdebf9fd8b1c5d132e1a423fb7477aa4b347
        (Note: Not included in Perl core distribution.)

      * fix spelling typo
        Tue Feb 24 19:49:03 2026 +0000
        d8b7ef4eb1507575598430c5e23e2cba95ca410e
---
 Porting/Maintainers.pl                            | 4 ++--
 cpan/Compress-Raw-Bzip2/lib/Compress/Raw/Bzip2.pm | 4 ++--
 cpan/Compress-Raw-Bzip2/t/000prereq.t             | 2 +-
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/Porting/Maintainers.pl b/Porting/Maintainers.pl
index 3c9645266b..87160c42fa 100755
--- a/Porting/Maintainers.pl
+++ b/Porting/Maintainers.pl
@@ -218,8 +218,8 @@ our %Modules = (
     },
 
     'Compress::Raw::Bzip2' => {
-        'DISTRIBUTION' => 'PMQS/Compress-Raw-Bzip2-2.213.tar.gz',
-        'SYNCINFO'     => 'jkeenan on Wed Aug 28 14:11:20 2024',
+        'DISTRIBUTION' => 'PMQS/Compress-Raw-Bzip2-2.218.tar.gz',
+        'SYNCINFO'     => 'jkeenan on Wed Mar 11 22:43:18 2026',
         'FILES'        => q[cpan/Compress-Raw-Bzip2],
         'EXCLUDED'     => [
             qr{^t/Test/},
diff --git a/cpan/Compress-Raw-Bzip2/lib/Compress/Raw/Bzip2.pm b/cpan/Compress-Raw-Bzip2/lib/Compress/Raw/Bzip2.pm
index ea9d38c55c..c2f9539f26 100644
--- a/cpan/Compress-Raw-Bzip2/lib/Compress/Raw/Bzip2.pm
+++ b/cpan/Compress-Raw-Bzip2/lib/Compress/Raw/Bzip2.pm
@@ -11,7 +11,7 @@ use Carp ;
 use bytes ;
 our ($VERSION, $XS_VERSION, @ISA, @EXPORT, $AUTOLOAD);
 
-$VERSION = '2.213';
+$VERSION = '2.218';
 $XS_VERSION = $VERSION;
 $VERSION = eval $VERSION;
 
@@ -384,7 +384,7 @@ See the Changes file.
 
 =head1 COPYRIGHT AND LICENSE
 
-Copyright (c) 2005-2024 Paul Marquess. All rights reserved.
+Copyright (c) 2005-2026 Paul Marquess. All rights reserved.
 
 This program is free software; you can redistribute it and/or
 modify it under the same terms as Perl itself.
diff --git a/cpan/Compress-Raw-Bzip2/t/000prereq.t b/cpan/Compress-Raw-Bzip2/t/000prereq.t
index 7d09746205..f63f0af74e 100644
--- a/cpan/Compress-Raw-Bzip2/t/000prereq.t
+++ b/cpan/Compress-Raw-Bzip2/t/000prereq.t
@@ -19,7 +19,7 @@ BEGIN
         if eval { require Test::NoWarnings ;  import Test::NoWarnings; 1 };
 
 
-    my $VERSION = '2.213';
+    my $VERSION = '2.218';
     my @NAMES = qw(
 
 			);
-- 
2.34.1

